internal/clients/sources/stubs/sources_stub.go

```diff
@@ -26,6 +32,46 @@ func (m *contextReadError) Error() string {
 	return "failed to find or convert dao stored in testing context"
 }
 
+// SourcesClientV2
+func WithSourcesClientV2(parent context.Context, init_store *[]sources.Source) context.Context {
+	ctx := context.WithValue(parent, sourcesCtxKey, &SourcesClientV2Stub{store: init_store})
+	return ctx
+}
+
+func getSourcesClientV2Stub(ctx context.Context) (si sources.ClientV2, err error) {
+	var ok bool
+	if si, ok = ctx.Value(sourcesCtxKey).(*SourcesClientV2Stub); !ok {
+		err = &contextReadError{}
+	}
+	return si, err
+}
+func (mock *SourcesClientV2Stub) GetArn(ctx context.Context, sourceId sources.ID) (string, error) {
+	return "", nil
+}
+
+func (mock *SourcesClientV2Stub) GetProvisioningTypeId(ctx context.Context) (string, error) {
+	return "", nil
```

Comment:
```suggestion
	return "1", nil
```

Or any number, really, we can be `42`

Reply:
Sure, I didn't "implement" it because those functions are not used yes :D 
but adding :)